### PR TITLE
Use `x86_64-unknown-linux-postgres` tuple

### PR DESCRIPTION
**Pull request by workingjubilee**
Introduces a much more secure implementation of `std::sys`.

### CODE REVIEW

**Comment by Hoverbear** [comment]
Ahh okay I see so this is bringing us closer to what "actual std" is and then specializing. I was wondering what all the additions were from!

File: run [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#! /usr/bin/sh
```
Comment (Hoverbear) [inline run:1]: I wonder if we can get a @BradyBonnette to help us make some CI for this stuff...

**Review by Hoverbear — APPROVED** [review]
Hard to detect the signal in the noise on what is being re-introduced and what is actually changed to be postgres specific, but it looks good overall.

Have you tested this from clean since the `run` script has been changed? (Update: I ran from clean and it seems fine)

**Comment by workingjubilee** [comment]
Yes I did in fact! And yes, some CI would Probably Help.

**Pull request merged**